_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -58,7 +58,7 @@ This rule assumes that the language of the [accessible name][] of each test targ
 
 ## Accessibility Support
 
-_There are no major accessibility support issues known for this rule._
+Some popular browser / screen reader combinations do not pronounce the accessible names of `svgs` elements. This can be resolved by adding an [explicit semantic role][] of `img` to the `svg` element.
```

Comment:
```suggestion
Some popular browser / screen reader combinations do not pronounce the accessible names of `svg` elements. This can be resolved by adding an [explicit semantic role][] of `img` to the `svg` element.
```